app/src/main/java/org/dosomething/letsdothis/ui/MainActivity.java

```diff
@@ -40,9 +46,27 @@ protected void onCreate(Bundle savedInstanceState)
                     .commit();
         }
 
+        initToolbar();
         initDrawer();
     }
 
+    private void initToolbar()
+    {
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        title = (TextView) findViewById(R.id.toolbar_title);
+        toolbar.setTitle("");
+        setSupportActionBar(toolbar);
+
+        final DrawerLayout drawerLayout = (DrawerLayout) findViewById(R.id.drawer_layout);
+        ActionBarDrawerToggle mDrawerToggle = new ActionBarDrawerToggle(this, drawerLayout, toolbar,
+                                                                        R.string.invite_code_opt,
+                                                                        R.string.account);
+        drawerLayout.setDrawerListener(mDrawerToggle);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+        getSupportActionBar().setHomeButtonEnabled(true);
+        mDrawerToggle.syncState();
+    }
+
     //TODO the drawer should have a list view, then we won't need all this logic
     private void initDrawer()
```

Comment:
TODO the drawer should have a list view, then we won't need all this redundant logic.